packages/www/src/pages/index.tsx

```diff
@@ -154,18 +152,17 @@ export default function Homepage() {
       <section class="brisa-section actions odd-section">
         <div class="code-example">
           <div class="info sticky">
-            <h2>📲 Browser-events on Server </h2>
+            <h2>📲 Browser-events on the server</h2>
             <p>
               Brisa mixes ideas from React's "<b>Server Actions</b>" and HTMX
               concepts. With Brisa, you can handle all browser events on the
-              server, such as forms, click events, etc. In addition, we offer
-              some extra <b>HTML attributes</b> to manage debounces, optimistic
-              updates, etc.
+              server, such as forms, click events and more. In addition, we
+              offer extra <b>HTML attributes</b> to manage debounces, optimistic
+              updates, among other things.
             </p>
             <p>
-              The idea is that if you want you can create a SPA without Web
-              Components, only with the weight of the Brisa RPC to make the
-              connection with the server.
+              You can create a SPA without Web Components, with just the payload
```

Comment:
On this one we need a bit more clarity. What the original is saying is that Brisa RPC has some js size, so client will have some Kb of JS in order to handle Brisa RPC connection.